### PR TITLE
add social icons from axach footer. 

### PR DESCRIPTION
**Pull request by danyball**
Fixex: 

- [x] - footer icons from axa.ch should be added at PL
- [x] - icons folder should be seperated from materials icons
- [x] - move internal-icons into icons 
- [x] replace all references of `@axa-ch/materials/icons`

# Merge Checklist:
- [x] Code selfreview has been performed ([Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)).
- [x] Tests are sufficient.
- [ ] Modifications still work in IE.
- [x] Documentation is up to date.
- [x] Changelog is up to date.
- [ ] Typescript typings for properties are up to date.
- [x] Designers approved changes or no approval needed.
- [x] Dependencies to other components still work.

### CODE REVIEW

File: src/components/00-materials/icons-raw/close.svg [hunk 1]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
\ No newline at end of file
+<svg width="24" height="24" viewBox="0 0 21 21" fill="none" xmlns="http://www.w3.org/2000/svg"><line x1="1.41421" y1="1" x2="8" y2="7.58579" stroke="#00008F" stroke-width="2" stroke-linecap="round"/><line x1="13.4142" y1="13" x2="20" y2="19.5858" stroke="#00008F" stroke-width="2" stroke-linecap="round"/><line x1="1" y1="19.5858" x2="19.5858" y2="1" stroke="#00008F" stroke-width="2" stroke-linecap="round"/></svg>
```
Comment (markus-walther) [inline src/components/00-materials/icons-raw/close.svg]: What's this change about?

File: src/components/00-materials/scripts/export-svg.js [hunk 2]
```diff
@@ -48,9 +50,12 @@ function readdirAndSaveSvgToJs(importPath, exportPath) {
         fileName = cleanFileName(fileName);
         const className = toClassName(fileName);
 
+        fs.mkdirSync(path.resolve(exportPath), {
+          recursive: true,
+        });
         // Generate a js file forEach svg file found
         fs.writeFileSync(
-          `${exportPath}/${fileName}.js`,
+          path.resolve(exportPath, `${fileName}.js`),
```
Comment (markus-walther) [inline src/components/00-materials/scripts/export-svg.js:58]: good!

**Review by markus-walther — DISMISSED** [review]
Looks good to me.

Just curious about that one change, an explanation is sufficient.

File: src/components/00-materials/icons-raw/material-design/close.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="currentColor" d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></svg>
```
Comment (raphaellueckl) [inline src/components/00-materials/icons-raw/material-design/close.svg:1]: What happened here?

File: src/components/10-atoms/icon/index.js [hunk 4]
```diff
@@ -13,8 +13,8 @@ import {
   SearchSvg,
   File_uploadSvg,
   Cloud_uploadSvg,
-} from '@axa-ch/materials/icons';
-import { arrowRightSvg, arrowLeftSvg } from '@axa-ch/materials/internal-icons';
+} from '@axa-ch/materials/icons/material-design';
+import { ArrowRightSvg, ArrowLeftSvg } from '@axa-ch/materials/icons';
```
Comment (raphaellueckl) [inline src/components/10-atoms/icon/index.js:17]: nice!

**Review by raphaellueckl — CHANGES_REQUESTED** [review]
very nice work, just one small confused comment.

File: src/components/00-materials/icons-raw/close.svg (hunk 1 above)
Comment (danyball) [reply src/components/00-materials/icons-raw/close.svg]: That was not a change. I have moved the material "close.svg" to the subfolder. And moved the internal-icon "close.svg" to the icons-raw folder. So it seems to be a change, but it is another icon.

**Review by raphaellueckl — DISMISSED** [review]

**Review by raphaellueckl — APPROVED** [review]

**Pull request merged**